packages/hs-react-ui/src/components/Button/__tests__/__snapshots__/Button.test.tsx.snap

```diff
@@ -158,10 +158,10 @@ exports[`Button shows ButtonContainer ButtonType.link 1`] = `
   -webkit-filter: drop-shadow(0em 0em 0em rgba(0,0,0,0.5));
   filter: drop-shadow(0em 0em 0em rgba(0,0,0,0.5));
   outline: 0 none;
-  border: 0 none;
+  border: 1px solid #252c35;
   cursor: pointer;
   background-color: transparent;
-  color: #252c35;
+  color: null;
```

Comment:
this doesn't seem right

Reply:
it's probably how it was before but we should change make sure it's not an invalid color